packages/storybook/src/schematics/configuration/root-files/.storybook/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This won't work. It's better not to have this base tsconfig. Your better off enforcing this into each individual app, otherwise, all stories will be included.

Reply:
This is for running a global storybook that will pick up all stories in the monorepo